core/templates/json_linked_data_for_event.template.php

```diff
@@ -26,7 +26,7 @@
   "description": <?php echo wp_json_encode($event_description); ?>,
   "url": "<?php echo esc_url_raw($event_permalink); ?>",
   "eventAttendanceMode": "<?php echo esc_url_raw('https://schema.org/' . $event_attendance_mode); ?>",
-  "eventStatus": [ <?php echo esc_html($event_status); ?> ],
+  "eventStatus": [ "<?php echo esc_html($event_status); ?>" ],
```

Comment:
since the content of that variable is a URL, this should be

```suggestion
  "eventStatus": [ "<?php echo esc_url_raw($event_status); ?>" ],
```

Reply:
Thanks, Brent.

I've no problem committing this but just checking you are aware that for [esc_url_raw()](https://developer.wordpress.org/reference/functions/esc_url_raw/) the codex has:

> This function is not safe to use for displaying the URL, use esc_url() instead.

Reply:
Yes, but this file is not used for display. It supplies event data for Google and other bots to read, therefore the URL needs to be functional. 

`esc_url()` will make the URL safe for display, but can potentially render it unusable depending on how it encodes things.

Reply:
If you wanted to both display a link AND keep it functional, you would use both `esc_url_raw()` and `esc_url()` like:

```php
<a href="<?php echo esc_url_raw( $url );?>"><?php echo esc_url( $url );?></a>
```

Reply:
Coolio ta, just double checkin.